atd-vze/src/views/Crashes/RelatedRecordsTable.js

```diff
@@ -76,7 +76,11 @@ const RelatedRecordsTable = ({
   const formatValue = (data, field) => {
     let fieldValue = data[field];
 
-    if (typeof data[field] === "object") {
+    // If the field comes from a hasura relationship then set the fieldValue accordingly
+    if (fieldConfig.fields[field].relationshipName) {
+      fieldValue = data[fieldConfig.fields[field].relationshipName][field];
```

Comment:
here is where we handle when a field comes from a hasura relationship and isnt a lookup table value

Reply:
very cool to see relationships being utilized here now 🙏